src/styles/blocks/stars--3.css

```diff
@@ -0,0 +1,44 @@
+.stars--3 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--3 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
You have a lot of repeated styles for the `.stars__star` class. To optimize your CSS, you can extract the common styles (display, background-image, width, height) and define them once for all `.stars--3 .stars__star` elements. Then, only specify the `margin-left` individually if they differ.